tests/regression/test_network_of_populations.py

```diff
@@ -58,6 +59,29 @@ def test_basic_simulation_with_dampening(data_api):
     _assert_baseline(result)
 
 
+def test_basic_simulation_stochastic(data_api):
+    network = np.createNetworkOfPopulation(
+        data_api.read_table("human/compartment-transition"),
+        data_api.read_table("human/population"),
+        data_api.read_table("human/commutes"),
+        data_api.read_table("human/mixing-matrix"),
+        data_api.read_table("human/infectious-compartments"),
+        data_api.read_table("human/infection-probability"),
+        data_api.read_table("human/initial-infections"),
+        data_api.read_table("human/trials"),
+        data_api.read_table("human/movement-multipliers"),
+        pandas.DataFrame([True], columns=["Value"]),
+        pandas.DataFrame([123], columns=["Value"])
+    )
+
+    result = calculateInfectiousOverTime(
+        np.basicSimulationInternalAgeStructure(network, 200, {"S08000016": {"[17,70)": 10}}),
+        network.infectiousStates,
+    )
```

Comment:
The reason I used this in previous tests was that this is what the basicSimulationInternalAgeStructure function used to return. However, given this is a brand new test, maybe we'd rather match the entire dataframe?